packages/ses/src/lockdown.js

```diff
@@ -284,6 +285,7 @@ export const repairIntrinsics = (options = {}) => {
   addIntrinsics(tameMathObject(mathTaming));
   addIntrinsics(tameRegExpConstructor(regExpTaming));
   addIntrinsics(tameSymbolConstructor());
+  shimArrayBufferTransfer();
```

Comment:
Wondering if it would make sense for "consistency" to have `shimArrayBufferTransfer()` return an empty "newIntrinsics" object, and have this line call `addIntrinsics()` like the other taming calls. While not needed for this specific case, conceptually a vetted shim may need to define new intrinsics, and I expect that future work to support vetted shims will require some API to do this registration.

Reply:
Done.

For the same "consistency", should I also rename `shimArrayBufferTransfer` to `tameArrayBufferTransfer`, and rename `shim-arraybuffer-transfer*` to `tame-arraybuffer-transfer*`?

My mild preference is not to.

Reply:
No, it's still a shim, it's installing a new thing instead of modifying an existing thing,

Reply:
Good. In that case, done.